crypto/key_manager.go

```diff
@@ -170,13 +179,14 @@ func parsePrivateKey(key []byte) (crypto.PrivateKey, trillian.SignatureAlgorithm
 			return nil, trillian.SignatureAlgorithm_ANONYMOUS, fmt.Errorf("unknown private key type: %T", key)
 		}
 	}
-	var err error
-	if key, err := x509.ParseECPrivateKey(key); err == nil {
-		return key, trillian.SignatureAlgorithm_ECDSA, nil
+	key2, err := x509.ParseECPrivateKey(key)
+	if err != nil {
+		glog.Warningf("error parsing EC key: %s", err)
+		return nil, trillian.SignatureAlgorithm_ANONYMOUS, errors.New("could not parse private key")
 	}
 
-	glog.Warningf("error parsing EC key: %s", err)
-	return nil, trillian.SignatureAlgorithm_ANONYMOUS, errors.New("could not parse private key")
+	return key2, trillian.SignatureAlgorithm_ECDSA, nil
+
```

Comment:
Happy to change it back, though when reading the code, this deviation from the [style guide](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow) caused me confusion